test/dailyLimitModule.js

```diff
@@ -5,7 +5,10 @@ const GnosisSafe = artifacts.require("./GnosisSafe.sol");
 const CreateAndAddModules = artifacts.require("./libraries/CreateAndAddModules.sol");
 const ProxyFactory = artifacts.require("./ProxyFactory.sol");
 const DailyLimitModule = artifacts.require("./modules/DailyLimitModule.sol");
-
+const MockContract = artifacts.require('./mocks/MockContract.sol');
+const MockToken = artifacts.require('./mocks/Token.sol');
+const abi = require('ethereumjs-abi');
```

Comment:
can we order the non-contract dependencies up?

Reply:
This has been done in the latest commit